src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseResultTestService.java

```diff
@@ -366,6 +370,41 @@ public void shouldGenerateTestwiseCoverageFileReports(Object resultNotification)
         assertThat(resultFromDatabase.getCoverageFileReportsByTestCaseName()).isNull();
     }
 
+    // Test
```

Comment:
what is the reason for those comments?

Reply:
These are common methods that are used as a test. They are not automated with @Test since they are called from two different other test-classes (Bamboo-Test and Jenkins test).

I just noticed that I'm right now only calling these tests once, I have to add the Jenkins tests as well. So thanks for your review!